app/src/sharedTest/java/org/oppia/android/app/topic/conceptcard/ConceptCardFragmentTest.kt

```diff
@@ -363,6 +370,112 @@ class ConceptCardFragmentTest {
     }
   }
 
+  @Test
+  fun testConceptCardFragment_launchSeveralConceptCardsWithSameSkill_onlyTheFirstOne() {
+    launchTestActivity().use { scenario ->
+      scenario.onActivity { activity ->
+        val fragment = ConceptCardFragment.bringToFrontOrCreateIfNew(
+          TEST_SKILL_ID_0,
+          ProfileId.getDefaultInstance(),
+          activity.supportFragmentManager
+        )
+        Assert.assertNotNull(fragment)
+        Assert.assertNull(
+          ConceptCardFragment.bringToFrontOrCreateIfNew(
+            TEST_SKILL_ID_0,
+            ProfileId.getDefaultInstance(),
+            activity.supportFragmentManager
+          )
+        )
+        Assert.assertEquals(1, activity.supportFragmentManager.fragments.size)
+        Assert.assertEquals(fragment, activity.supportFragmentManager.fragments[0])
```

Comment:
Prefer using Truth assertions over JUnit's (and statically import them). Ditto for the other tests.

Reply:
Done